management-interface.go

```diff
@@ -125,18 +125,26 @@ func getRaspberryPiSerialNumber() string {
 
 // Return the salt minion ID for the device.
 func getSaltMinionID() string {
+	return strings.TrimSpace(readFile("/etc/salt/minion_id"))
+}
+
+// Return the time of the last salt update.
+func getLastSaltUpdate() string {
+	return strings.TrimSpace(readFile("/etc/cacophony/last-salt-update"))
```

Comment:
Following on from the suggested change in the other PR, you'll want to parse and then nicely format the timestamp.